resources/translations/de_DE/troubleshooting/layer_splitting.md

```diff
@@ -0,0 +1,45 @@
+
+Um Warping in höheren Schichten zu verhindern, kann man einige Dinge tun, um die Wahrscheinlichkeit einer Aufspaltung der Schicht zu verringern:
+* Verwenden Sie ein konzentrisches [Unteres/oberes Muster](../top_bottom/top_bottom_pattern.md). Ein lineares Muster hat Linien, die alle in dieselbe Richtung schrumpfen, was die Schichten in dieselbe Richtung zieht. Das konzentrische Muster verteilt diese Kraft.
+* Die Erhöhung der [Temperatur der Druckabmessung](../material/build_volume_temperature.md) verringert die Verformung durch [Aushärtung](https://en.wikipedia.org/wiki/Annealing_%28glass%29) des Kunststoffs. Dazu benötigt der Drucker ein geschlossenes Bauvolumen, in dem die Wärme gehalten wird.
+* Auch die Verwendung eines [Windschutzes](../experimental/draft_shield_enabled.md) hält einen Teil der Wärme im Inneren, wenn auch weniger effektiv als ein geschlossenes Bauvolumen.
+* Verhindern Sie scharfe Ecken im 3D-Modell. Das Hinzufügen einer Rundung an einer scharfen Ecke verhindert, dass die Linien der Außenhaut direkt an die darunter liegenden Wände stoßen. Dies ist oft eine Stelle, an der die Schicht aufgespalten wird.
+* Verwenden Sie ein Material, das nicht so stark schrumpft. PLA zum Beispiel verzieht sich weniger als Polypropylen. Die meisten Menschen wählen ihr Material jedoch nach bestimmten Anforderungen aus, so dass dies für sie nicht in Frage kommt.
+
+Unzureichender Druck
+----
+Damit die Schichten gut aneinander haften, ist es hilfreich, den Kunststoff stark auf die vorherige Schicht zu drücken. Wenn die Düse nicht genug Material ausstößt oder das Material nicht stark genug gegen die vorherige Schicht gedrückt wird, verringert sich die Haftkraft zwischen den Schichten, so dass sie sich leichter trennen können. Auch wenn dies nicht sofort zu einem Riss in der Schicht führt, verringert sich dadurch die Festigkeit des fertigen Teils.
+
+Dies ist vor allem ein Problem für die Wände. Die Außenhaut hat eine so große Oberfläche, die nicht zum Reißen neigt. Zwischen Füllung und Wänden sorgen die Wände für den größten Teil der Schichthaftung, da die Füllung oft sehr schnell gedruckt wird, mit spärlich verteilten Linien und oft nicht vertikal gestapelt.
+
+Um einen ausreichenden Druck zu gewährleisten, muss sichergestellt werden, dass keine [Unterextrusion](underextrusion.md) vorliegt. Es gibt alle Ursachen für Unterextrusion, aber einige davon sind für das Layer Splitting von besonderer Bedeutung:
+* Wenn man den Druck auf halber Strecke unterbricht, dauert es eine Weile, bis die Extrusion wieder in Gang kommt. Das "Skript Pause at Height" hat eine Option, um <!--if cura_version >= 4.7-->eine Schicht zu wiederholen<!--endif--><!--if cura_version < 4.7:redo the last few layers-->. Dadurch wird das Material vorbereitet, bevor es nach oben weitergeht, und eine schwache Schicht mit unzureichendem Düsendruck verhindert.
```

Comment:
Missing a translation here for `redo the last few layers`.
Feel free to restructure the sentence if the word order of German doesn't allow it to split off like this.

Reply:
fixed